src/applications/vre/28-8832/config/chapters/static-information/staticInformationComponent.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
This component is re-used so often, it'd definitely make a good candidate for general use in formation-react, or `platform/`

Reply:
I agree, at some point we should make some tickets for this